### ISSUE TITLE
Improve project sharing copy

### ISSUE THREAD

**Issue by xmatthewx** [mozilla/webmaker-core#946]
When a user shares a project, Webmaker sends out 

> Check out this project on Webmaker! https://.... 

The share would be more inspiring and links more likely to be followed if we used the project title. Most mainstream platforms don't name themselves in their shares. 

Proposed: Check out `TITLE` by `USERNAME`. 

This is executed in webmaker-[android /util/Share.java](https://github.com/mozilla/webmaker-android/blob/47b70eea3774fceea2d9bebfec95bb3a6725b9aa/app/src/main/java/org/mozilla/webmaker/util/Share.java).

**Comment by xmatthewx** [comment]
opinions @vazquez on this?

Proposed: Check out `TITLE` by `USERNAME`.

**Comment by ryanwarsaw** [comment]
@vazquez update?

**Comment by xmatthewx** [comment]
You can just run with the proposed change @ryanw-se. Thanks!

**Comment by ryanwarsaw** [comment]
@xmatthewx I looked into doing a hyperlink but I don't think that will work very well since some apps don't support it, instead I opted-in for "Check out `<title>` by `<username>`. `<link>`"

**Comment by xmatthewx** [comment]
Yep. That makes sense. That's what I was thinking but didn't make clear. `URL` at the end.

**Comment by ryanwarsaw** [comment]
Quick update on this, I did start work on this a few weeks back but got blocked because of my limited knowledge on how we send data to the backend on this (webmaker-core). I plan to do a bit more research on this soon and knock this out.

Dev note: From preliminary testing, I've determined the easiest way to do this would be to utilize the Memory Storage API, however this is only temporary until we switch over to the Queue based system.
